test/Common.cpp

```diff
@@ -289,6 +289,15 @@ bool isValidSemanticTestPath(boost::filesystem::path const& _testPath)
 	return true;
 }
 
+boost::unit_test::precondition::predicate_t minEVMVersionCheck(langutil::EVMVersion _minEVMVersion)
+{
+	auto predicate = [_minEVMVersion](boost::unit_test::test_unit_id) {
```

Comment:
Any reason for not returning the predicate directly? It's gonna get RVOed anyway, so not a big deal regardless.

Reply:
Ah, good point, I can do that :) Did it like this just by the force of habit where I first define the function and then use it.